README.md

```diff
@@ -1,2 +1,11 @@
 # cheri-examples
 cheri-riscv sample c programs
+
+## Build instructions with CMake
+1. Create a separate build directory (\<build-dir\>) outside the source directory (\<source-dir\>).
+2. Generate the build files using either **Unix Makefiles** or **Ninja**. Currently only the *RISC-V* cheri-purecap toolchain file has been created. From outside the source directory execute: 
```

Comment:
We need to wrap long lines (in general: either to 80 or 100 chars as you prefer; but if the file/repo has already set a standard, let's stick with it), to help `diff` in the future (and our future selves too!).